browser-fetching/index.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: layouts/base.njk
+title: How to fetch data using a web browser
+eleventyNavigation:
+  key: Fetch
+  order: 4
+---
+
+# Aren't you fetching?!
+
+_I'm uncomfortable, you're probably uncomfortable, let's move on to fetching in the web browser_.
+
+All modern web browsers supply a Fetch API for making a HTTP requests to a web server, and handling the data returned from the server. The Fetch API provides a programming interface to asynchronously exchange data with a web server without needing to load another web page.
```

Comment:
small grammar error here - "a HTTP requests"

Reply:
Thank you.